src_c/rect.c

```diff
@@ -980,15 +971,20 @@ pg_rect_contains(pgRectObject *self, PyObject *arg)
     return PyBool_FromLong(ret);
 }
 
-static int
+static PyObject *
```

Comment:
the return type should not change here, because `pg_rect_contains_seq` is a "slot" function so the expected return type is an integer itself. The `ret` from `_pg_rect_contains` must be returned directly without `PyBool_FromLong`